src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapePlayer.java

```diff
@@ -141,6 +143,26 @@ public boolean canBeArrested() {
         return _isWanted || _inRestrictedArea;
     }
 
+//	########################################
+//	#                Money                 #
+//	########################################
+
+    public int getBalance() {
+        return _balance;
+    }
+
+    public void setBalance(int balance) {
+        _balance = balance;
+    }
+
+    public void increaseBalance(int missionPay) {
+        _balance += missionPay;
+    }
+
+    public void decreaseBalance(int itemPrice) {
```

Comment:
I would prefer takeMoney, but if you don't want to change the name that's also ok.